src/Stripe.net/Services/TransferReversals/TransferReversalService.cs

```diff
@@ -21,48 +21,50 @@ public TransferReversalService(string apiKey)
         {
         }
 
+        public override string BasePath => "/transfers/{PARENT_ID}/reversals";
+
         public bool ExpandBalanceTransaction { get; set; }
 
         public bool ExpandTransfer { get; set; }
 
         public virtual TransferReversal Create(string transferId, TransferReversalCreateOptions options = null, RequestOptions requestOptions = null)
         {
-            return this.Post($"{Urls.BaseUrl}/transfers/{transferId}/reversals", requestOptions, options);
+            return this.CreateNestedEntity(transferId, options, requestOptions);
         }
 
         public virtual Task<TransferReversal> CreateAsync(string transferId, TransferReversalCreateOptions options = null, RequestOptions requestOptions = null, CancellationToken cancellationToken = default(CancellationToken))
         {
-            return this.PostAsync($"{Urls.BaseUrl}/transfers/{transferId}/reversals", requestOptions, cancellationToken, options);
+            return this.CreateNestedEntityAsync(transferId, options, requestOptions, cancellationToken);
         }
 
         public virtual TransferReversal Get(string transferId, string reversalId, RequestOptions requestOptions = null)
         {
-            return this.GetEntity($"{Urls.BaseUrl}/transfers/{transferId}/reversals/{reversalId}", requestOptions);
+            return this.GetNestedEntity(transferId, reversalId, requestOptions);
```

Comment:
I see you used the version without `null` for options. but you did not use it everywhere. I know I kinda asked for it but per my previous comment I'm starting to think being explicit with what params we pass and why is likely safer/better after all (soooorry)